### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
removed dead/broken links from the document per Sirkkuw

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2991

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Comment by Sirkkuw** [comment]
@yogkumgit - please add this one back in:
1.	How to register for their additional verification method - https://support.microsoft.com/office/ace1d096-61e5-449b-a875-58eb3d74de14
It has been fixed

**Comment by msbemba** [comment]
@Sirkkuw  I have added this one back to the document. Please merge the PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/3031

Thanks
Sri

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -92,12 +92,11 @@ If you have previously turned on per person MFA, you must turn it off before ena
 1. On the multi-factor authentication page, select each user and set their Multi-Factor auth status to **Disabled**.
 
 ## Next steps
-
 - [How to register for their additional verification method](https://support.microsoft.com/office/ace1d096-61e5-449b-a875-58eb3d74de14)
+- [What is: Multifactor Authentication](https://support.microsoft.com/en-us/help/4577374/what-is-multifactor-authentication)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
- [What is: Multifactor Authentication](https://support.microsoft.com/en-us/help/4577374/what-is-multifactor-authentication)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Please remove the locale from the added link.

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]: Thank you.I have removed the locale from the URL

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Proposed changes implemented, please merge this PR. Thanks.

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3017

**Pull request merged**